Library/Formula/bltool.rb

```diff
@@ -0,0 +1,15 @@
+class Bltool < Formula
+  desc "A tool for command-line interaction with backloggery.com"
+  homepage "https://github.com/ToxicFrog/bltool"
+  url "https://github.com/ToxicFrog/bltool/releases/download/v0.2.1/bltool-0.2.1.zip"
+  sha256 "dceb5812058699b726bcd43a87c703eba39527959f712ef262919e0e14d6608e"
+
+  def install
+    libexec.install "bltool.jar"
+    bin.write_jar_script libexec/"bltool.jar", "bltool"
+  end
+
+  test do
+    assert shell_output("#{bin}/bltool --help formats").include? "backloggery"
```

Comment:
I'd rather make this `assert_match /backloggery/, shell_output("#{bin}/bltool --help formats")`. Does the same thing - Just more explicitly and it's the style we're dominantly using recently.